public/controllers/management/management.js

```diff
@@ -272,8 +275,11 @@ export class ManagementController {
     this.breadCrumbBack();
   }
 
-  switchFilesSubTab(flag) {
+  switchFilesSubTab(flag,showFile) {
     this.managingFiles = flag || true;
+    if(showFile){
```

Comment:
Why we need this `if`? why not directly `this.showFile = showFile` ?

Reply:
Because `showFilesSubtab` can be called from different locations, so, sometimes the `showFile` param could arrive empty. It's the expected behavior because we only want to set up the variable if the param is different from undefined.